rotkehlchen/tests/api/test_erc20_info.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import requests
+
+from rotkehlchen.chain.ethereum.typing import string_to_ethereum_address
+from rotkehlchen.tests.utils.api import api_url_for, assert_proper_response_with_result
+
+
+@pytest.mark.parametrize('number_of_eth_accounts', [0])
+def test_query_token_with_info(rotkehlchen_api_server):
+    """Query DAI token to retrieve basic information"""
+
+    response = requests.get(
+        api_url_for(
+            rotkehlchen_api_server,
+            'erc20tokeninfo',
+        ), json={
+            'address': string_to_ethereum_address('0x6B175474E89094C44Da98b954EedeAC495271d0F'),
+        },
+    )
+
+    result = assert_proper_response_with_result(response)
+    assert result['decimals'] == 18
+    assert result['symbol'] == 'DAI'
+    assert result['name'] == 'Dai Stablecoin'
+
+    # Test a contract without decimals/name/symbol methods
+    response_2 = requests.get(
+        api_url_for(
+            rotkehlchen_api_server,
+            'erc20tokeninfo',
+        ), json={
+            'address': string_to_ethereum_address('0x7d655c57f71464B6f83811C55D84009Cd9f5221C'),
+        },
+    )
+
+    result_2 = assert_proper_response_with_result(response_2)
+    assert len(result_2.keys()) == 3
+    assert all((value is None for value in result_2.values()))
```

Comment:
You should add a 3rd test here also for an EOA address. As I tested above it does not seem to work, and raises an exception. Which results in a 500 error.

Reply:
Sorry m y fault. In the previous version I catched this error, assumed that this version worked the same way and forgot to double check this situation. As you mention adding a test should have been the first thing. Done now.